app/models/court_date.rb

```diff
@@ -35,6 +37,14 @@ def additional_info?
   def display_name
     "#{casa_case.case_number} - Court Date - #{I18n.l(date.to_date)}"
   end
+
+  private
+
+  def set_court_report_due_date
+    if date.present? && court_report_due_date.blank?
```

Comment:
I added a conditional for `court_report_due_date.blank?`. Without this, the value entered in the Court Report Due Date in the UI won't have any affect